calico-vpp-agent/watchers/net_watcher.go

```diff
@@ -41,6 +44,7 @@ type NetworkDefinition struct {
 	Name              string
 	LoopbackSwIfIndex uint32
 	Range             string
+	Nad               string
```

Comment:
Maybe `NetAttachDefs` ? It's a bit longer, but it does spare the hesitation "what's a NAD ?"